server/index.js

```diff
@@ -143,40 +159,40 @@ app.put('/updateItem', (req, res) => {
   const newInfo = req.body;
   const id = newInfo.product_id;
 
-  if (!id) {
+  if (!id) {  //|| 0
     console.log('product_id required to update item');
     res.status(404).send()
   } else {
-    Overview.updateOne({product_id: id}, newInfo)
-    .then(doc => {
-      const productInfo = doc;
-      console.log(`Success updating item ${id}`);
-      res.send(productInfo);
-    })
-    .catch(err => {
-      console.log(`error updating item ${id}: `, err);
-      res.status(404).send(err);
-    })
+    Overview.updateOne({ product_id: id }, newInfo) //have a separate validation func for put/post?
```

Comment:
Good idea! (In reference to your comment about having a separate data validation func)